test/src/docmanager/widgetmanager.spec.ts

```diff
@@ -0,0 +1,310 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import expect = require('expect.js');
+
+import {
+  ServiceManager, utils
+} from '@jupyterlab/services';
+
+import {
+  IMessageHandler, Message, sendMessage
+} from 'phosphor/lib/core/messaging';
+
+import {
+  Widget
+} from 'phosphor/lib/ui/widget';
+
+import {
+  DocumentRegistry, TextModelFactory, ABCWidgetFactory, Context
+} from '../../../lib/docregistry';
+
+import {
+  DocumentWidgetManager
+} from '../../../lib/docmanager';
```

Comment:
Could you swap the import order for `'../../../lib/docregistry'` and `'../../../lib/docmanager'`?